OPA.ps1

```diff
@@ -0,0 +1,55 @@
+#Requires -Version 5.1
+<#
+    .SYNOPSIS
+        This script installs the required OPA executable used by the
+        assessment tool
+    .DESCRIPTION
+        Installs the OPA executable required to support SCuBAGear.
+    .EXAMPLE
+        .\OPA.ps1
+#>
+# Set prefernces for writing messages
+$DebugPreference = "Continue"
+$InformationPreference = "Continue"
+$ErrorActionPreference = "Stop"
+
+# Set expected version and OutFile path
+$ExpectedVersion = "0.42.1"
+$InstallUrl= "https://openpolicyagent.org/downloads/v$($ExpectedVersion)/opa_windows_amd64.exe"
+$OutFile=(Join-Path $pwd.Path $InstallUrl.SubString($InstallUrl.LastIndexOf('/')))
+$ExpectedHash ="5D71028FED935DC98B9D69369D42D2C03CE84A7720D61ED777E10AAE7528F399"
+
+# Download files
+try {
+    Write-Information "Downloading $InstallUrl"
+    $WebClient = New-Object System.Net.WebClient
+    $WebClient.DownloadFile($InstallUrl, $OutFile)
+    Write-Information ""
+    Write-Information "`nDownload of `"$OutFile`" finished."
+}
+catch {
+    "An error has occurred: Unable to download OPA executable, try manual install see details in README"
+}
+$WebClient.Dispose()
+
+# Hash checks
+if ((Get-FileHash .\opa_windows_amd64.exe).Hash -eq $ExpectedHash)
+    {
+    Write-Information "SHA256 verified successful"
+    }
+else {
+    Write-Information "SHA256 verified failed, try re-download or manual install see details in README "
+}
+# Version checks
+$InstalledVersion= .\opa_windows_amd64.exe version | Select-Object -First 1
```

Comment:
This may fail if the organization is using aggressive binary allowlisting, so I think it probably is best to wrap it in a try/catch and provide a helpful error message.